extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -18,90 +18,97 @@ describes.realWin(
     },
   },
   (env) => {
-    let win;
-    let element;
-    let shoppingConfig;
-    let storeService;
+    let pageElement;
 
     beforeEach(async () => {
-      win = env.win;
-      storeService = getStoreService(win);
-      registerServiceBuilder(win, 'story-store', function () {
-        return storeService;
-      });
-
-      await createAmpStoryShoppingConfig();
+      pageElement = <amp-story-page id="page1"></amp-story-page>;
+      env.win.document.body.appendChild(pageElement);
     });
 
-    async function createAmpStoryShoppingConfig() {
-      const pageEl = win.document.createElement('amp-story-page');
-      pageEl.id = 'page1';
-      element = createElementWithAttributes(
-        win.document,
-        'amp-story-shopping-config',
-        {'layout': 'nodisplay'}
+    async function createAmpStoryShoppingConfig(
+      src = null,
+      config = defaultConfig
+    ) {
+      pageElement.appendChild(
+        <amp-story-shopping-config layout="nodisplay" src={src}>
+          <script type="application/json">{JSON.stringify(config)}</script>
+        </amp-story-shopping-config>
```

Comment:
We don't need this to be backwards compatible since it hasn't been released yet.
We can remove the `amp-story-shopping-config` completely.

Reply:
As discussed offline, I'll let you make that change after this merges. Merely a matter of changing the query selector and this test.